.travis.yml

```diff
@@ -71,51 +71,53 @@ matrix:
 
   allow_failures:
     - python: 3.5
-      TESTS: py3status
+      env: TESTS=py3status
```

Comment:
Yes, this is a fix.  If you run the .travis.yml file under https://lint.travis-ci.org/ you get an error.
See:
https://github.com/buildbot/buildbot/pull/2669